.gitignore

```diff
@@ -41,4 +41,6 @@ jspm_packages
 
 dist
 public/
+src/server/usersInfo/
```

Comment:
Why usersInfo?! If we keep fles there it shoukd be moved to sercer/static/users/